src/scripts/googleApi.ts

```diff
@@ -26,100 +30,171 @@ export async function getPlaylistDetails(
 
 export const mockedVideoConfig = [
   {
+    index: 1,
     title:
-      "p2p, rpc: Manual block-relay-only connections with addnode #24170",
-    img: "https://i.ytimg.com/vi/4YCUHsfFazQ/default.jpg",
-    link: "https://www.youtube.com/watch?v=4YCUHsfFazQ",
-  },
-  {
-    title: "Demo 2: A Tour Around Bitcoin Core Codebase",
-    img: "https://i.ytimg.com/vi/MbinzItqsXI/default.jpg",
-    link: "https://www.youtube.com/watch?v=MbinzItqsXI",
-  },
-  {
-    title: "Demo 1: Compiling and Testing a PR.",
-    img: "https://i.ytimg.com/vi/n5CRJRqkAoc/default.jpg",
-    link: "https://www.youtube.com/watch?v=n5CRJRqkAoc",
-  },
-  {
-    title:
-      "p2p: Allow whitelisting outgoing connections #27114",
-    img: "https://i.ytimg.com/vi/EYPfGw9Z14w/default.jpg",
-    link: "https://www.youtube.com/watch?v=EYPfGw9Z14w",
-  },
-  {
-    title:
-      "p2p: track AddrMan totals by network and table, improve precision of adding fixed seeds #26847",
-    img: "https://i.ytimg.com/vi/fp5lmVss--Q/default.jpg",
-    link: "https://www.youtube.com/watch?v=fp5lmVss--Q",
-  },
-  {
-    title:
-      "Activation logic for testing consensus changes #16",
-    img: "https://i.ytimg.com/vi/gXPKYZujeJE/default.jpg",
-    link: "https://www.youtube.com/watch?v=gXPKYZujeJE",
-  },
-  {
-    title:
-      "Wallet: Add foreign_outputs metadata to support CoinJoin transactions #25991",
-    img: "https://i.ytimg.com/vi/oE3pQMK1oTk/default.jpg",
-    link: "https://www.youtube.com/watch?v=oE3pQMK1oTk",
-  },
-  {
-    title:
-      "Introduce secp256k1 module with field and group classes to test framework #26222",
-    img: "https://i.ytimg.com/vi/jdRbTmekF8U/default.jpg",
-    link: "https://www.youtube.com/watch?v=jdRbTmekF8U",
-  },
-  {
+      "The Thriving Career of Bitcoin Development in India | Coin Selection | Summer of Bitcoin | E01",
+    description:
+      "In this episode, we talk with Anmol Sharma. Anmol is an\n" +
+      " - IIIT Jabalpur student and 3rd-year Electronics and Telecom engineering\n" +
+      "- SOB alumn and  FOSS contributor to [bcoin](https://twitter.com/bcoin)\n" +
+      "- working as a Bitcoin Engineer in [Bold](https://twitter.com/Bold).\n" +
+      "\n" +
+      "In this episode, we covered\n" +
+      " - His Background and Introduction to SOB\n" +
+      " - His journey into the rabbit hole.\n" +
+      " - His work with @bcoin on coin-selection\n" +
+      " - The emerging Indian Bitcoin Dev community.\n" +
+      " - Required developer resources and materials.\n" +
```

Comment:
Not entirely sure, but are the video data coming from here instead of the api? because i noticed the video list on the pr review club got changed too!

Reply:
This is mock data just for dev purposes so that we don’t unnecessarily call the api multiple times. The API is only called when we deploy the website to bitshala.github.io. You can see this logic in the review page how videoConfig is calculated. The isProd variable checks if its local or production.